xchangelog

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
I think this could be just sh

Reply:
I made this `/bin/bash` again because xbps-src templates are written in Bash, and I actually had an error once when I ran `xchangelog` on a template with some Bashism (can't remember the exact package).